docs/getting-started/symfony-bundle.md

```diff
@@ -305,13 +304,13 @@ class HomePage extends AbstractController
         </div>
     {% endif %}
 
-    {% if is_enabled('some_feature', identity) %}
+    {% if 'some_feature' is enabled_for(identity) %}
         <div>
           <p>This section is only visible with "some_feature" enabled for request located in "barcelona"</p>
         </div>
     {% endif %}
     
-    {% if is_enabled('in_progress_feature', identity) %}
+    {% if is_feature_enabled('in_progress_feature', app.user ? app.user.id : 'anon', {location: app.user ? app.user.location : app.request.query.get('location', 'unknown'), role: app.user ? app.user.role : 'IS_AUTHENTICATED_ANONYMOUSLY'} %}
```

Comment:
Here, I want to show the ability to build an identity object from the function call.